torch/_dynamo/symbolic_convert.py

```diff
@@ -1085,15 +1085,32 @@ def WITH_CLEANUP_FINISH(self, inst):
         self.popn(2)
         self.push(None)
 
+    def CALL_FINALLY(self, inst):
+        """
+        pushes the address of the next instruction onto the stack and increments
+        bytecode counter by delta
+        """
+        # Python 3.8 only
+        assert self.next_instruction is not None
+        addr = self.indexof[self.next_instruction]
+        self.push(ConstantVariable.create(addr))
+        self.instruction_pointer = self.indexof[inst.target]
+
     def END_FINALLY(self, inst):
+        # Python 3.8 only
+        # https://docs.python.org/3.8/library/dis.html#opcode-END_FINALLY
         tos = self.pop()
-        assert tos is None
+        if isinstance(tos, ConstantVariable):
+            self.instruction_pointer = tos.as_python_constant()
+        else:
+            pass
 
     def POP_FINALLY(self, inst):
+        # Python 3.8 only
         preserve_tos = inst.argval
         if preserve_tos:
             tos = self.pop()
-        assert self.pop() is None
+        _ = self.pop()
```

Comment:
Pls don't remove the assertion, do sth like:
```
if sys.version_info == (3, 8):
  _ = self.pop()
else:
  assert self.pop() is None
```

Reply:
This opcode only exists in Python 3.8 ([ref](https://github.com/python/cpython/blob/9afb0e1606cad41ed57c42ea0a53ac90433f211b/Misc/NEWS.d/3.9.0a2.rst#L237)). Thus, I think it is safe to remove the assertion.

Reply:
Have something with a side-effect in an assert statement is bad style anyway (because those assert statements can be ommitted entirely if you pass the right args to Python).